### ISSUE TITLE
Remote Resource UserUserPreferencesResourceImpl is not correctly configured for autologging.

### ISSUE THREAD

**Issue by stroomdev10** [gchq/stroom#3358]
Also the Stroom User event that is emitted is not schema compliant.

it has <Update/> for EventDetail

**Comment by at055612** [comment]
Now returns a valid event
```xml
    <User>                                                                                                                                                                                                                                                                      
      <Id>admin</Id>                                                                                                                                                                                                                                                            
    </User>                                                                                                                                                                                                                                                                     
  </EventSource>                                                                                                                                                                                                                                                                
  <EventDetail>                                                                                                                                                                                                                                                                 
    <TypeId>UserPreferencesResourceImpl.update</TypeId>                                                                                                                                                                                                                         
    <Description>Updating own user preferences</Description>                                                                                                                                                                                                                    
    <Update>                                                                                                                                                                                                                                                                    
      <Before>                                                                                                                                                                                                                                                                  
        <Object>                                                                                                                                                                                                                                                                
          <Type>UserPreferences</Type>                                                                                                                                                                                                                                          
          <Description>UserPreferences</Description>                                                                                                                                                                                                                            
          <Data Name="dateTimePattern" Value="yyyy-MM-dd'T'HH:mm:ss.SSSXX"/>                                                                                                                                                                                                    
          <Data Name="density" Value="Compact"/>                                                                                                                                                                                                                                
          <Data Name="editorKeyBindings" Value="VIM"/>                                                                                                                                                                                                                          
          <Data Name="editorTheme" Value="tomorrow_night"/>                                                                                                                                                                                                                     
          <Data Name="font" Value="Roboto"/>                                                                                                                                                                                                                                    
          <Data Name="fontSize" Value="Medium"/>                                                                                                                                                                                                                                
          <Data Name="theme" Value="Dark"/>                                                                                                                                                                                                                                     
          <Data Name="timeZone">                                                                                                                                                                                                                                                
            <Data Name="use" Value="UTC"/>                                                                                                                                                                                                                                      
            <Data Name="id"/>                                                                                                                                                                                                                                                   
            <Data Name="offsetHours"/>                                                                                                                                                                                                                                          
            <Data Name="offsetMinutes"/>                                                                                                                                                                                                                                        
          </Data>                                                                                                                                                                                                                                                               
        </Object>                                                                                                                                                                                                                                                               
      </Before>                                                                                                                                                                                                                                                                 
      <After>                                                                                                                                                                                                                                                                   
        <Object>                                                                                                                                                                                                                                                                
          <Type>UserPreferences</Type>                                                                                                                                                                                                                                          
          <Description>UserPreferences</Description>                                                                                                                                                                                                                            
          <Data Name="dateTimePattern" Value="yyyy-MM-dd'T'HH:mm:ss.SSSXX"/>
          <Data Name="density" Value="Compact"/>
          <Data Name="editorKeyBindings" Value="STANDARD"/>
          <Data Name="editorTheme" Value="tomorrow_night"/>
          <Data Name="font" Value="Roboto"/>
          <Data Name="fontSize" Value="Medium"/>
          <Data Name="theme" Value="Dark"/>
          <Data Name="timeZone">
            <Data Name="use" Value="UTC"/>
            <Data Name="id"/>
            <Data Name="offsetHours"/>
            <Data Name="offsetMinutes"/>
          </Data>
        </Object>
      </After>
    </Update>
  </EventDetail>
```

**Comment by at055612** [comment]
Fixed in > 7.2-beta.18
